test/factories/users.rb

```diff
@@ -17,6 +17,7 @@
       svie_member_type { 'RENDESTAG' }
       after(:build) do |user|
         user.svie_primary_membership = create(:membership, user: user).id
+        user.primary_membership.group.update(issvie: true)
```

Comment:
Instead of updating the group here it'd nicer to add a `:for_svie_group` trait to the membership factory
```ruby
trait :for_svie_group
  association :group, issvie: true # or it can be added to the group factory as a trait and used from here
end
```
and here you can use
```ruby
user.svie_primary_membership = create(:membership, :for_svie_group, user: user).id
```